docs/deployment/linux.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Deployment on a Linux Server
+---
+
+Deploying Velocity on a Linux machine is easy! This guide will walk you through the installation of Velocity.
+
+Basic Linux knowledge is assumed.
+
+# Tools
+
+This is a glossary of all the tools and commands we will be using throughout the guide.
+
+These tools are available on most Linux distributions, and many come pre-installed with them. If you aren't sure if they're installed, run the command. If you see `command not found`, you'll need to install them from your package manager.
+
+## GNU Screen
+
+When connecting to a server through SSH, you are limited to running one process at a time. If the SSH session were to disconnect, or you want to run a process in the background, `screen` allows us to do so. We will use `screen` to run the Velocity proxy even if we disconnect from the server.
+
+If you're more comfortable with `tmux`, then feel free to use that instead.
+
+**Commands**
+- `screen` creates a screen session
+- `screen -S proxy` creates a screen session with the ID 'proxy'
+- <kbd>CTRL+A+D</kbd> disconnects you from your current screen session
+- `screen -r` connects you to the running screen session
+- `screen -r proxy` connects you to the screen session with the ID 'proxy'
+
+## GNU nano
+
+`nano` is a basic user-friendly text editor. If you prefer to use an alternative such as `vim` or `emacs`, go right ahead :]
+
+**Commands**
+- `nano` opens up the nano text editor
+- `nano file.txt` edits (or creates if it doesn't exist) a file named 'file.txt'
+- <kbd>CTRL+S</kbd> saves the file
+- <kbd>CTRL+X</kbd> exits `nano` (will ask if you want to save if you haven't yet)
+
+# Setup
+
+## Setting up your Workspace
+
+To keep things nice and clean, create a folder to store your Velocity server in. The name and location doesn't matter, but make sure you have read, write, and execute permissions in this folder.
+
+Enter the directory using `cd folder_name`.
+
+## Downloading Velocity
+
+You can find the latest version [on the downloads site](https://www.velocitypowered.com/downloads). Download the latest version to your newly created folder (or use FTP to upload it).
+
+As an example, I'll use this command:
+```shell script
+wget -O proxy.jar https://ci.velocitypowered.com/job/velocity/246/artifact/proxy/build/libs/velocity-proxy-1.0.11-SNAPSHOT-all.jar
```

Comment:
Even if temporary, you can replace this by https://ci.velocitypowered.com/job/velocity-1.1.0/lastSuccessfulBuild/artifact/proxy/build/libs/velocity-proxy-1.1.0-SNAPSHOT-all.jar

Reply:
Done 👍